r2/r2/lib/menus.py

```diff
@@ -609,7 +605,7 @@ def make_title(self, attr):
 class SearchSortMenu(SortMenu):
     """Sort menu for search pages."""
     _default = 'relevance'
-    mapping = search_sorts
+    mapping = g.search.sorts
```

Comment:
Trying to get to this today, got some day-job stuff to take care of first.

On Tue, May 5, 2015 at 2:25 PM, Keith Mitchell <notifications@github.com>
wrote:

> In r2/r2/lib/menus.py
> <https://github.com/reddit/reddit/pull/1289#discussion_r29714715>:
>
> > @@ -609,7 +605,7 @@ def make_title(self, attr):
> >  class SearchSortMenu(SortMenu):
> >      """Sort menu for search pages."""
> >      _default = 'relevance'
> > -    mapping = search_sorts
> > +    mapping = g.search.sorts
>
> I've found that VMenu.param_docs relies on looking up _options from the
> class, rather than an instance of the class (which more or less makes
> sense, since param_docs runs at load time.
>
> This of course complicates the situation and means I don't have a good
> solution to this tangled web of premature access to pylons.g just yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reddit/reddit/pull/1289/files#r29714715>.
>